### DISCUSSION TITLE
readinessProbe/livenessProbe for kubernetes-external-secrets deployment

### DISCUSSION THREAD

**Discussion by ajitksingh1** [external-secrets/kubernetes-external-secrets#594] [General]
Hey KES Team,

In the kubernetes-external-secrets deployment using helm chart, readinessProbe/livenessProbe is not available in deployment.yaml template.  Wondering what could be the reason for not adding readinessProbe/livenessProbe in deployment template ?

**Comment by gabegorelick** [comment]
See https://github.com/external-secrets/kubernetes-external-secrets/issues/266 and https://github.com/external-secrets/kubernetes-external-secrets/issues/485

**Comment by gabegorelick** [comment]
I filed another issue for this (https://github.com/external-secrets/kubernetes-external-secrets/issues/863).
